File: app/src/main/java/com/theappsolutions/boilerplate/util/other_utils/BuildInfoUtils.java
package com.theappsolutions.boilerplate.util.other_utils;

import android.content.Context;
import android.content.pm.PackageManager;

/**
 * @author Dmytro Yakovlev d.yakovlev@theappsolutions.com
 * @copyright (c) 2017 TheAppSolutions. (https://theappsolutions.com)
 */
public class BuildInfoUtils {

    public static int getVersionCode(Context context) {
        int v = 0;
        try {
            v = context.getPackageManager().getPackageInfo(context.getPackageName(), 0).versionCode;
        } catch (PackageManager.NameNotFoundException ignored) {

        }
        return v;
    }
}

File: app/src/main/java/com/theappsolutions/boilerplate/util/other/BuildInfoUtils.kt
package com.theappsolutions.boilerplate.util.other

import android.content.Context
import android.content.pm.PackageManager

/**
 * @author Dmytro Yakovlev d.yakovlev@theappsolutions.com
 * @copyright (c) 2017 TheAppSolutions. (https://theappsolutions.com)
 */
fun getVersionCode(context: Context): Int {
    var v = 0
    try {
        v = context.packageManager.getPackageInfo(context.packageName, 0).versionCode
    } catch (ignored: PackageManager.NameNotFoundException) {

    }
    return v
}
